apps/alerts/main.py

```diff
@@ -34,7 +54,52 @@ def index(request):
         if request.path == "/":
             return main_html
 
-    except:
+        if request.path.endswith("list_exams"):
+            return jsonify(db.collection("exam-alerts").document("all").get().to_dict()["exam-list"])
+
+        if request.path.endswith("fetch_data"):
+            exam = request.json["exam"]
+            email = get_email(request)
+            student_data = db.collection("exam-alerts").document(exam).collection("students").document(email).get().to_dict()
+            announcements = list(db.collection("exam-alerts").document(exam).collection("announcements").stream())
+            return jsonify({
+                "success": True,
+                "exam_type": "ok-exam",
+                "questions": [
+                    {
+                        "questionName": question["student_question_name"],
+                        "startTime": question["start_time"],
+                        "endTime": question["end_time"],
+                    }
```

Comment:
why inconsistent names

Reply:
Javascript is camelCase, python is snake_case

Reply:
when the student app receives data, it's the questionName as far as it knows, since it never sees any other name